gnovm/stdlibs/std/native.go

```diff
@@ -59,7 +59,8 @@ func findPrevFuncName(m *gno.Machine, targetIndex int) string {
 			return string(currFunc.Name)
 		}
 	}
-	panic("function name not found")
+
+	return ""
```

Comment:
What are the implications of us not panicking here? @thehowl

Reply:
Wait, I missed all of this findPrevFunc/TargetFunc dance. Why are we doing this? Why are we not simply iterating from `m.NumFrames()-3` until we find a frame with a function?

The implication: looks here like there was a case where the panic was being hit, @r3v4s can you enlighten us? As for the implication, if anything this should cause the event to have an empty name, as this is the only place where we use this fn

Reply:
if we panic instead of returning "", calling Bar() will panic

https://github.com/gnolang/gno/blob/252577668f81e62062e24f63f2feca98ea60dccc/gno.land/cmd/gnoland/testdata/event_normal.txtar#L14-L19

https://github.com/gnolang/gno/blob/252577668f81e62062e24f63f2feca98ea60dccc/gno.land/cmd/gnoland/testdata/event_normal.txtar#L54-L59

Reply:
And iteration from `m.NumFrames()-3` doesn't find SubFoo as prev func name during here.
https://github.com/gnolang/gno/blob/252577668f81e62062e24f63f2feca98ea60dccc/gno.land/cmd/gnoland/testdata/event_normal.txtar#L14-L19

https://github.com/gnolang/gno/blob/252577668f81e62062e24f63f2feca98ea60dccc/gno.land/cmd/gnoland/testdata/event_normal.txtar#L33-L45

Reply:
Is this change necessary if we remove PrevFunc?

Reply:
I don't think so

Reply:
revert dd17a9686d03aee1cf18cc4d67a253f01266fc5d